src/lib/utils/find.ts

```diff
@@ -0,0 +1,13 @@
+function find<TItem>(
```

Comment:
maybe add a comment why we don't use native find always?

```suggestion
// We aren't using the native Array.find, because introducing the refactor away from lodash in a minor/patch
// adding a new required polyfill at this step, which before was only required for niche use-cases
// was decided as too risky. 
// @MAJOR  this can be replaced by native Array#find
function find<TItem>(
```